README.md

```diff
@@ -1,17 +1,31 @@
 # NodeList.forEach polyfill
 
-MDN polyfill for the `NodeList.forEach` method.
-
-It polyfills `forEach` support for `NodeList` objects (a common result of `document.querySelectorAll`).
+Provides a polyfill for [Nodelist.prototype.forEach()](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach) to all Browsers supporting [ES5](https://caniuse.com/#search=es5).
 
 ## Native support
 
-Chrome 51, Firefox 50, Opera 38, Safari 10
+Chrome 51, Firefox 50, Opera 38, Safari 10<br>

```

Comment:
In Markdown you can use double space for new lines (I should've done this - good catch)